app/scripts/views/mixins/experiment-mixin.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'underscore',
+  'lib/experiment',
+  'lib/url'
+], function (_, ExperimentInterface) {
+  'use strict';
+
+  return {
+    initialize: function (options) {
+      options = options || {};
+
+      this.notifications = options.notifications;
+
+      this.experiments = new ExperimentInterface({
+        window: this.window,
+        able: options.able,
+        metrics: this.metrics,
+        account: this._account,
+        user: options.user,
+        notifications: options.notifications
+      });
+
+      this.experiments.chooseExperiments();
+    },
+
+    isOptedInToExperiment: function (experimentName) {
```

Comment:
Maybe rename this to `isInExperiment` or the other function to `isOptedInToExperimentGroup`